ui/design-system-core/src/components/ModeratorDetailCard/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import dayjs from 'dayjs';
+import { tw } from '@twind/core';
+
+import { Moderator } from '@akashaorg/typings/ui';
+
+import SocialLink from './social-link';
+import Avatar from '../Avatar';
+import Text from '../Text';
+import Tooltip from '../Tooltip';
+
+export interface IModeratorDetailCardProps {
+  moderator: Moderator;
+  hasBorderBottom: boolean;
+  tenureInfoLabel: string;
+  onSocialLinkClick: () => void;
+}
+
+const ModeratorDetailCard: React.FC<IModeratorDetailCardProps> = props => {
+  const { moderator, hasBorderBottom, tenureInfoLabel, onSocialLinkClick } = props;
+
+  const formatDate = (date: string) => {
+    return dayjs(date).format('MMM YYYY');
+  };
+
+  const truncateStyle =
+    'max-w-[12.5rem] md:max-w-[7.5rem] w-fit whitespace-nowrap overflow-hidden text-ellipsis cursor-default';
+
+  return (
+    <div
+      className={tw(
+        `flex py-4 flex-none ${
+          hasBorderBottom ? 'border-b-1 border-solid border-grey8 dark:border-grey3' : ''
```

Comment:
use grouping syntax to make it more readable border(b-1 solid grey8 dark:grey-3)